server/src/lib.rs

```diff
@@ -260,6 +260,11 @@ async fn mirror(
 ) -> Result<warp::reply::Response, Rejection> {
     let (path, query, method, headers, body) = request;
 
+    // Handle load balancer health check with a simple response
+    if method == Method::GET {
+        return Ok("healthy".into_response());
```

Comment:
Is there a dependency on this endpoint returning "healthy" string? Otherwise we can simply return successful no content response.

This might do the trick
```rust
Response::builder()
    .status(StatusCode::NO_CONTENT)
```

Reply:
`NO_CONTENT` works better. `healthy` is just industry standard, but no need for any specific string.